hack/make-rules/test-e2e-node.sh

```diff
@@ -148,7 +148,7 @@ else
 
   # Do not use any network plugin by default. User could override the flags with
   # test_args.
-  test_args='--kubelet-flags="--network-plugin= --network-plugin-dir=" '$test_args
+  test_args='--kubelet-flags="--network-plugin= --cni-bin-dir=" '$test_args
```

Comment:
I think this is for local testing where no network plugin is used. But it should be able to do it without specifying the network-plugin flag. @Random-Liu confirm?